soc/espressif/common/loader.c

```diff
@@ -216,13 +211,12 @@ void map_rom_segments(uint32_t app_drom_start, uint32_t app_drom_vaddr,
 				CACHE_DROM_MMU_MAX_END - cache_mmu_irom_size);
 #endif
 	/* Show map segments continue using same log format as during MCUboot phase */
-	BOOT_LOG_INF("DROM segment: paddr=%08xh, vaddr=%08xh, size=%05Xh (%6d) map",
+	ESP_EARLY_LOGI(TAG, "DROM segment: paddr=%08xh, vaddr=%08xh, size=%05Xh (%6d) map",
 			app_drom_start_aligned, app_drom_vaddr_aligned,
 			app_drom_size, app_drom_size);
-	BOOT_LOG_INF("IROM segment: paddr=%08xh, vaddr=%08xh, size=%05Xh (%6d) map\r\n",
+	ESP_EARLY_LOGI(TAG, "IROM segment: paddr=%08xh, vaddr=%08xh, size=%05Xh (%6d) map\r\n",
 			app_irom_start_aligned, app_irom_vaddr_aligned,
```

Comment:
You may save some flash by using the exact same format string  `"%s segment: paddr= ..."`